x-pack/metricbeat/module/gcp/constants.go

```diff
@@ -84,6 +85,7 @@ const (
 	StorageResourceLabelLocation  = "resource.label.location"
 	CloudSQLResourceLabelRegion   = "resource.labels.region"
 	DataprocResourceLabelLocation = "resource.label.region"
+	RedisResourceLabelRegion      = "resource.label.region"
```

Comment:
Should we change the one above to `DataprocResourceLabelRegion` to match the name or maybe use `ResourceLabelRegion` for both dataproc and redis?

Reply:
Can we move forward with merging this PR and keep this discussion going, as is not directly related to the code changes in this PR?

What do you think about removing any reference to `Location` or `Region` from these constants? I don't think it adds much value and like in this case is prone to errors.

Reply:
Im ok either way, keeping Location/Region/Zone or not, just want to be consistent here.

Reply:
https://github.com/elastic/beats/pull/33892 I just created this PR as a followup to the comment here so we don't forget to do it :)